src/background/logging.ts

```diff
@@ -315,13 +315,13 @@ export async function _setLoggingConfig(config: LoggingConfig): Promise<void> {
 export const getLoggingConfig = liftBackground(
   "GET_LOGGING_CONFIG",
   async () => {
-    return await _getLoggingConfig();
+    return _getLoggingConfig();
   }
 );
 
 export const setLoggingConfig = liftBackground(
   "SET_LOGGING_CONFIG",
   async (config: LoggingConfig) => {
-    return await _setLoggingConfig(config);
+    return _setLoggingConfig(config);
   }
```

Comment:
An additional fix that this change highlights, would be to drop the wrapper function in many cases, but this might change the behavior slightly.
```suggestion
  _getLoggingConfig
```

But it can be made safe again with

```suggestion
  ary(_getLoggingConfig, 1)
```

Want this change too, where possible? In many cases this avoids having to duplicate types in this wrapper function

Reply:
I have a slight readability preference for `unary` vs. `ary`

I think I prefer the more verbose arrow function since it's less error prone if you forget to wrap with unary. But Typescript would potentially catch most errors of forgetting to wrap in unary?

Reply:
Naked function references are probably unsafe, as I learned recently in a discussion with Addy Osmani if I remember correctly. There's a related rule for this, but it's only for arrays: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md

We can circle back to these once I start looking at the messaging